src/Components/Button.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Button = ({typeOfButton, clickHandler, toggleDisabled, buttonName}) => {
+    return (
+        <button
+        type={typeOfButton}
+        onClick={clickHandler}
+        disabled={toggleDisabled}
+        >{buttonName}</button>
```

Comment:
Ik zou `{buttonName}` en `</button>` beiden op een eigen regel zetten. Dat is net wat overzichtelijker.